### DISCUSSION TITLE
Please tell me your favorite drinks.

### DISCUSSION THREAD

**Discussion by srboc** [NAIST-SE/AttractingContributors#923] [2023 Attraction Contributions]
https://github.com/srboc/AttractingContributors

**Comment by bconfessor** [comment]
I like Iyemon Green Tea, I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by Gloria109** [comment]
i adore mimt tea. it always helps me relax.
